### PR TITLE
fix: api-package edit form bug

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
fix api-package edit form bug.

## Does this PR introduce a user interface change?
- [ ] Yes(screenshot is required)
- [x] No

## Which versions should be patched?
release/1.1

## Which issue(s) this PR fixes:

Fixes # api-package edit form bug

### CODE REVIEW

File: shell/app/modules/msp/pages/gateway/containers/api-package-create.tsx [hunk 1]
```diff
@@ -87,7 +80,11 @@ const BindDomainForm = (props: any) => {
               },
             ]}
           >
-            <Input className="bind-domain-input" onChange={(e: object) => changeItemValue(e.target.value, index)} />
+            <Input
+              className="bind-domain-input"
+              value={value[index]}
+              onChange={(e: object) => changeItemValue(e.target.value, index)}
```
Comment (McDaddy) [inline shell/app/modules/msp/pages/gateway/containers/api-package-create.tsx:86]: e is not object

File: shell/app/modules/msp/pages/gateway/containers/api-package-create.tsx [hunk 2]
```diff
@@ -308,7 +301,7 @@ export const PureApiPackage = () => {
           }
         }}
       >
-        <RenderPureForm form={basicForm} {...formProps} list={fieldsList} />
+        <RenderPureForm form={basicForm} {...formProps} list={fieldsList} onlyItems />
```
Comment (McDaddy) [inline shell/app/modules/msp/pages/gateway/containers/api-package-create.tsx:304]: will `onlyItems` cause issue?

Comment (hujiahao-hjh) [reply shell/app/modules/msp/pages/gateway/containers/api-package-create.tsx:304]: Because it has a Form around it, only need Items inside.

**Review by daskyrk — APPROVED** [review]

**Pull request merged**